ambari-server/src/test/resources/stacks/HDP/0.1/services/PIG/metainfo.xml

```diff
@@ -34,6 +34,16 @@
             <timeout>600</timeout>
           </commandScript>
         </component>
+        <component>
+          <name>SOME_CLIENT_FOR_SERVICE_CHECK</name>
```

Comment:
This extra component should be accounted for in this assertion:

https://github.com/apache/ambari/blob/04110f721437520eed50b1023e6ddda617a272d1/ambari-server/src/test/java/org/apache/ambari/server/stack/StackManagerTest.java#L246

Reply:
Thanks for noticing